docs/releasehistory.rst

```diff
@@ -6,15 +6,22 @@ Release History
 
 Changed behaviors
 -----------------
--   ``LangevinDynamicsMove`` now uses ``openmm.LangevinMiddleIntegrator`` (a BAOAB integrator) instead of `openmm.LangevinIntegrator`` (an OBABO integrator).
+- ``LangevinDynamicsMove`` now uses ``openmm.LangevinMiddleIntegrator`` (a BAOAB integrator) instead of ``openmm.LangevinIntegrator`` (an OBABO integrator).
```

Comment:
@ijpulidos : I think we also wanted to fix the formatting of this one, because its appearing as bolded and italicized

Reply:
I think I fixed this in the commit below